parsec/core/gui/claim_device_widget.py

```diff
@@ -613,7 +613,7 @@ def _on_retrieve_info_error(self, job: QtToTrioJob[None]) -> None:
         assert self.dialog is not None
         self.dialog.reject()
 
-    def _on_page_failed(self, job: QtToTrioJob[None]) -> None:
+    def _on_page_failed(self, job: Optional[QtToTrioJob[None]]) -> None:
```

Comment:
use `QtToTrioJob[None] | None` instead

Reply:
Done in [b48dcaf](https://github.com/Scille/parsec-cloud/pull/3628/commits/b48dcafc97a1486872522e71667e23bf303e73bb)